### ISSUE TITLE
How to change sort to Ascending or Descending?

### ISSUE THREAD

**Issue by komputerboy** [Dimibe/grouped_list#5]
Hi, i want to sort by date Descending.

My code is like this:
```
GroupedListView<dynamic, String>(
      groupBy: (element) => element.day,
      elements: _motionList,
      sort: true,
      groupSeparatorBuilder: (String value) => Padding(
        padding: const EdgeInsets.all(8.0),
        child: Center(
            child: Text(
          value,
          style: TextStyle(fontSize: 20, fontWeight: FontWeight.bold),
        )),
      ),
      itemBuilder: (c, element) {
        return Card(
          elevation: 8.0,
          margin: new EdgeInsets.symmetric(horizontal: 10.0, vertical: 6.0),
          child: Container(
            child: ListTile(
              contentPadding:
                  EdgeInsets.symmetric(horizontal: 20.0, vertical: 10.0),
              leading: Icon(Icons.account_circle),
              title: Text(DateFormat.Hm()
                  .format(DateTime.fromMillisecondsSinceEpoch(element.time))),
              trailing: Icon(Icons.arrow_forward),
            ),
          ),
        );
      },
    );
```

and i got the output is:
```
12/16/2019
     - 10:14
     - 11:14
12/17/2019
     - 08:11
     - 09:10
```
and how to reverse it, so the data will be like this:
```
12/17/2019
     - 09:10
     - 08:11
12/16/2019
     - 11:14
     - 10:14
```
thank you

**Comment by Dimibe** [comment]
Currently there is no option to reverse the sorting. I'll put this on my todo list. 
For the moment you could  set `sort` to `false` and sort your  `_motionList` before passing it to the GroupedListView Widget.

**Comment by Dimibe** [comment]
Resolved and released in version 1.3.0. Have a look at the docs for more more detailed information about the usage.
